### DISCUSSION TITLE
Tortoise multi-gpu support?

### DISCUSSION THREAD

**Discussion by chris480** [rsxdalv/TTS-WebUI#189] [Q&A]
Is it possible to get TortoiseTTS to use multi-gpus or any settings to speed things up?

**Comment by rsxdalv** [comment]
It should give decent results. Tortoise improves quality by generating more, so if you split the generations 50:50 then you could get a good speedup.

I haven't seen a multi GPU implementation myself, but it's *not that hard on paper*, you just need to get the developer time to do it.

**Comment by rsxdalv** [comment]
For now I'd just recommend running two UIs in parallel. Might need some extra stuff to select GPU 2.
